lib/standalone.mjs

```diff
@@ -250,7 +256,12 @@ export default async function standalone({
 				/**
 				 * If we're fixing, save the file with changed code
 				 */
-				if (postcssResult.root && postcssResult.opts && !postcssResult.stylelint.ignored && fix) {
+				if (
+					postcssResult.root &&
+					postcssResult.opts &&
+					!postcssResult.stylelint.ignored &&
+					Boolean(autofix)
```

Comment:
In the same way as https://github.com/stylelint/stylelint/pull/8106#discussion_r1844933564, I think that this change, [which use `config.fix`](https://github.com/stylelint/stylelint/pull/8106/files#diff-a9211bfb8f0453e9d6e413fe613dbdcd388621ae34881d7400c93f10dafe516bR121), we should keep `&& fix` if it doesn't break your tests.
i.e. have a separate PR that takes into account `config` and `configFile`

Reply:
Thanks, good catch! Fixed it.
[fix: config is not yet supported](https://github.com/stylelint/stylelint/pull/8106/commits/ac7866f9cc0fb21df72353367841279bdbe03c52)